build.gradle

```diff
@@ -33,7 +31,7 @@ javafx {
     modules = [ 'javafx.controls', 'javafx.fxml' ]
 }
 
-mainClassName = 'server.Application' //'GUIStarter'
```

Comment:
we should default to this imho, spring boot was just an experiment/addition, not the main functionality, though we could add a separate command for it

Reply:
as in default to Main